api/src/modules/auth/authentication.service.ts

```diff
@@ -81,9 +92,67 @@ export class AuthenticationService {
     };
   }
 
-  async logIn(user: User): Promise<UserWithAccessToken> {
+  private async createBackOfficeSession(
+    user: User,
+    accessToken: string,
+  ): Promise<BackOfficeSession> {
+    // We replicate what adminjs does by default using postgres as session storage (the default in memory session storage is not production ready)
+    // This implementation is not compatible with many devices per user
+    await this.backOfficeSessionRepository
```

Comment:
Nice. Maybe it would be nice to move this to some private method like:
`clearExistingSessionAdminSession` or smth like that.

It also feels weird to not declaratively know what would happen if the query does not target any row but I guess we can live with that as long as we know how the underlying query works